src/LibP4est.jl

```diff
@@ -7426,11 +7014,13 @@ end
 
 Transform an edge across one of the adjacent faces into a neighbor tree. This version expects the neighbor face and orientation separately.
 
+`.`
```

Comment:
What is this?

Reply:
This is strange and connected to your other comment, I guess. For some reason the `.` is moved upwards, but it is "only" inside a docstring, so should not destroy anything severely. Looking at the corresponding [docstring in `p4est`](https://github.com/cburstedde/p4est/blob/aafb87d93e33dffe24d67533d56bc2f0cdb72605/src/p8est_connectivity.h#L429) it looks like Clang.jl has some problems parsing the `\a` with more than one character that is supposed to be italic.

Reply:
I try to create an MWE.

Reply:
OK, thanks for the explanation. It's not a big issue here, but maybe it can be reported upsteam (to Clang.jl?). Either way, I wouldn't consider it blocking here

Reply:
I found the issue here. This was actually a mistake in the docstring from `p4est` and was fixed in a [commit](https://github.com/cburstedde/p4est/commit/4507aebb6e2663894cca3b396282f466b50746e0) that is not included in the `p4est` version we are currently wrapping. This should be fixed if we switch to a newer p4est version, see https://github.com/trixi-framework/P4est.jl/issues/76. I checked that Clang.jl parses the fixed docstring correctly.